components/members/main/MemberDetail/index.tsx

```diff
@@ -92,19 +99,23 @@ const MemberDetail: FC<MemberDetailProps> = ({ memberId }) => {
                 </NameWrapper>
                 <div className='intro'>{profile?.introduction}</div>
               </div>
-              <ContactWrapper>
-                <Link passHref href={`tel:${profile?.email}`} legacyBehavior>
-                  <div style={{ cursor: 'pointer' }}>
-                    <CallIcon />
-                    <div className='phone'>{profile?.phone}</div>
-                  </div>
-                </Link>
-                <Link passHref href={`mailto:${profile?.email}`} legacyBehavior>
-                  <div style={{ cursor: 'pointer' }}>
-                    <MailIcon />
-                    <div className='email'>{profile?.email}</div>
-                  </div>
-                </Link>
+              <ContactWrapper shouldDivide={!isEmptyField(profile?.phone) && !isEmptyField(profile?.email)}>
```

Comment:
p3; 그냥 이렇게 할 수도 있지 않았을까요!?

```suggestion
              <ContactWrapper shouldDivide={!!profile?.phone && !!profile?.email}>
```

Reply:
헉!! 그러네요 👍